pyccel/codegen/printing/fcode.py

```diff
@@ -2112,6 +2112,51 @@ def _print_PyccelFloorDiv(self, expr):
                 code = 'real({})'.format(code)
         return code
 
+    def _print_PyccelRShift(self, expr):
+        args = [self._print(a) for a in expr.args]
+        code = args[0]
+        for c in args[1:]:
+            code = 'RSHIFT({}, {})'.format(code, c)
```

Comment:
Have you tested whether `args` can ever have a length > 2 ? As `RShift` is a binary operator I am not sure that this is possible. If it isn't then the loop can be removed

Reply:
I tested it and as you said args can't exceed a length of 2